packages/maskbook/package.json

```diff
@@ -90,11 +90,13 @@
     "react-use": "^15.3.4",
     "react-virtualized-auto-sizer": "^1.0.2",
     "react-window": "^1.8.6",
+    "snapshot.js": "^0.1.0",
```

Comment:
Can you explain what is this? https://www.npmjs.com/package/snapshot.js I cannot understand the meaning of this lib

Reply:
lol the real one: https://www.npmjs.com/package/@snapshot-labs/snapshot.js